backend/backend/properties.py

```diff
@@ -253,6 +253,53 @@ def _is_in_monopoly(self):
         return is_monopoly
 
 
+def property_positions():
+    """Get a list of board positions where there are properties
+
+    Returns:
+        A list representing the positions of all properties on the board.
+
+    """
+    conn = backend.storage.make_connection()
+    try:
+        conn.begin()
+        with conn.cursor() as cursor:
+            cursor.execute('SELECT property_position '
+                           'FROM property_values;')
+            result = [row["property_position"] for row in cursor.fetchall()]
+        return result
+    finally:
+        conn.close()
+
+
+def is_property_owned(property_position, game_id):
+    """Check if a property is currently owned.
+
+    Arguments:
+        property_position: An int representing the property position on the
+                           board.
+        game_id: An int representing the game id of the game in question.
+
+    Returns:
+        A bool: True if property is owned, False otherwise.
+
+    """
+    conn = backend.storage.make_connection()
+    try:
+        conn.begin()
+        with conn.cursor() as cursor:
+            cursor.execute('SELECT * '
+                           'FROM properties '
+                           'WHERE state = "owned" '
+                           'AND property_position = %s '
+                           'AND game_id = %s;', (property_position, game_id))
+
+            return bool(cursor.rowcount > 0)
```

Comment:
Do you need to call `bool` here?

Reply:
It was suggested by pylint as an alternative to a fully-fledged if-statement. I think it looks fairly neat to be honest.

Reply:
Gaby just pointed this out and now I realise this is what you probably meant: The `bool()` function didn't need to be there, instead just have `cursor.rowcount > 0`. Apologies for the mis-understanding.

Reply:
No apologies needed, I should have been clearer! It hadn't occurred to me that the simpler form wouldn't have been obvious.